core/src/main/java/com/linecorp/armeria/common/logging/RequestContextExporterBuilder.java

```diff
@@ -75,35 +88,72 @@ public boolean containsBuiltIn(BuiltInProperty property) {
      */
     @Deprecated
     public Set<BuiltInProperty> getBuiltIns() {
-        return ImmutableSet.copyOf(builtIns);
+        return builtIns.stream().map(entry -> entry.key).collect(toImmutableSet());
     }
 
     /**
      * Adds the specified {@link AttributeKey} to the export list.
+     * The specified {@code alias} prefixed with {@value PREFIX_ATTRS} is used for the export key.
+     * This method is a shortcut for {@code addAttribute(alias, attrKey, true)}.
```

Comment:
How about making a breaking change here? We could remove `addAttribute(..., boolean)` and never prepend `"attrs."` for simplicity. `"attrs."` could be prepended only when a user used `"attrs.foo:...."`.

Reply:
Agreed, You're telling me. :-) I tried to avoid breaking changes but the final API seems weird.
Let me address it.